specify/specify7
# Add tectonic default ranks

Fixes #5294

### Checklist

      and self-explanatory (or properly documented)

### Testing instructions

- Open Tectonic Tree
- Verify the ranks below are present
- - [ ] Superstructure
- - [ ] Tectonic domain
- - [ ] Tectonic subdomain
- - [ ] Tectonic unit
- - [ ] Tectonic subunit
- [ ] Verify there is a root node
